### PR TITLE
Consolidate location to postprocessor methods to keep api in sync

### PR DESCRIPTION
**Pull request by andrewleech**
The api endpoint for process_method is currently missing the ability to trigger a postprocess with reversed symlink due to it having a whitelist that doesn't include this method.

This pull request fixes this by consolidating the list of available methods in postProcessor.py and uses the same method names for processing and api. 

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by OmgImAlexis** [comment]
Please change the files permissions back to 644 instead of the 755 you've got them set to in the commit.

**Review by sharkykh — APPROVED** [review]
LGTM. Thanks

**Comment by andrewleech** [comment]
Yeah sorry about the devolop branch, I was going to fix that but then saw develop and master was in sync and hoped it wouldn't matter... 
@OmgImAlexis thanks for the permissions pick up I completely missed that, was working remotely at the time.

File: sickbeard/postProcessor.py [hunk 1]
```diff
@@ -1178,7 +1178,7 @@ def process(self):  # pylint: disable=too-many-return-statements, too-many-local
                     raise EpisodePostProcessingFailedException("File is locked for reading/writing")
                 self._moveAndSymlink(self.file_path, dest_path, new_base_name, sickbeard.MOVE_ASSOCIATED_FILES,
                                      sickbeard.USE_SUBTITLES and ep_obj.show.subtitles)
-            elif self.process_method == METHOD_REVERSE_SYMLINK:
+            elif self.process_method == METHOD_SYMLINK_REVERSED:
```
Comment (sharkykh) [inline sickbeard/postProcessor.py]: It's a good thing you caught that... lol

Comment (andrewleech) [reply sickbeard/postProcessor.py]: yeah tell me about it, and I call myself a professional..... sometimes....

**Pull request merged**